apps/landingPage/src/components/header.js

```diff
@@ -0,0 +1,67 @@
+// @flow
+
+import React from 'react';
+import { Link } from 'gatsby';
+import { Heading } from '@kiwicom/orbit-components/lib/';
+import styled from 'styled-components';
+
+// $FlowFixMe - Cannot resolve module RelativeImageStub ( possible solution https://bit.ly/2BYfHRr )
+import logo from '../images/logo.png';
+
+const links = [
+  { title: 'About Margarita', link: '/' },
+  { title: 'Playground', link: '/playground' },
+  { title: 'Features', link: '/features' },
+  { title: 'Demo', link: '/demo' },
+  {
+    title: 'Installation',
+    link: 'https://github.com/kiwicom/margarita/',
+  },
+];
+
+export default () => (
+  <Header>
+    <a href="https://code.kiwi.com/">
+      <Logo src={logo} alt="Logo" />
+    </a>
+    <HeaderRight>
+      {links.map(el => {
+        const internal = /^\/(?!\/)/.test(el.link);
```

Comment:
Is that necessary? Does Gatsby Link only work for internal links?

Reply:
Yes it works only with internal, but anyway I think I ll change this part